python/pyspark/pandas/tests/test_dataframe_splits/split_s_x.py

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import unittest
+
+
+if __name__ == "__main__":
+    from pyspark.pandas.tests.test_dataframe import *  # noqa: F401
+
+    try:
+        import xmlrunner
+
+        testRunner = xmlrunner.XMLTestRunner(output="target/test-reports", verbosity=2)
+    except ImportError:
+        testRunner = None
+
+    loader = unittest.loader.TestLoader()
+    loader.testNamePatterns = [
+        "pyspark.pandas.tests.test_dataframe.DataFrameTests.test_[stuvwxyz]*",
```

Comment:
Can we also include `y` and `z` here?

Reply:
it include `y` and `z` in this pattern `...test_[stuvwxyz]*`

Reply:
Yeah, I mentioned it because the file name indicates from s to x, but actual pattern allows s to z. Not a big deal, tho. :-)

Reply:
yes, it should be name `s_z`. Sorry I misunderstand your comment.